.uberenv_config.json

```diff
@@ -2,11 +2,12 @@
 "package_name" : "serac",
 "package_version" : "develop",
 "package_final_phase" : "initconfig",
-"package_source_dir" : "../..",
+"package_source_dir" : ".",
```

Comment:
I'm pretty sure this was an unintentional change uberenv.  Do you know the line that caused this?

Reply:
It looks like this was a result of a change related to https://github.com/LLNL/uberenv/issues/44

With the current (old) version of uberenv we use, hostconfigs end up in the directory from which we run uberenv/build_tpls (the repo root) and in the TPL directory (/usr/WS2/smithdev/libs/serac/blueos_3_ppc64le_ib_p9/2021_04_21_14_18_07 for example).  It seems that placing hostconfigs in the repo tree is undesirable (per the above issue) and was removed by [this line](https://github.com/LLNL/uberenv/pull/46/files#diff-815c7c31845f6d60a7744f452038d9aacb5b2614e3005e5401ea3ecb0c395951R640), which uses `package_source_dir` relative to the `--prefix` and not the directory from which uberenv was run.